Add emotes to Emotes.png and Emotes.json

okliis opened the pull request:
Added 5 new emotes:

- pepeLaugh
- Prayge
- PogU
- NODDERS
- pressF

codecat commented:
Thanks for the PR! A couple things:

* The frames for `pepeLaugh` are a lot of wasted space. This emote as far as I can tell is just the emote going up and down. This is doable by only having 1 frame in the sheet with some transparency above it.
* `NODDERS` currently occupies 4 frames in the sheet but it could be 3 (the 4th frame is a repeat of the 2nd frame)
* The last 3 frames in `pressF` are repeated frames as well, they can be omitted.

Also, I do not want this version of `PogU`. The guy in the picture (Ryan Gutierrez) is a far-right conspiracy theorist who is anti-vax and supports a lot of other garbage. (This is the same reason that Twitch themselves changed `PogChamp`.) Maybe there's another emote that can replace it?

okliis commented:
Hey,
Ah, I had no idea about `PogU`, I'll get rid of it ASAP, thanks for letting me know! 
Regarding `pepeLaugh`, `NODDERS` and `pressF` : good point, I'll optimize them right away.

okliis commented:
Alright, so:

1. I removed `PogU` and replaced it with `WOWERS` (way better tbh). 
2. `pepeLaugh` is now just 1 frame + alpha.
3. `NODDERS` is now 3 frames.
4. `pressF` is now 5 frames.
5. `Prayge` was left as is.

Much neater. Tested locally and all seem to work well!
Let me know if you'd like more added, it'd be a pleasure.

codecat commented:
Thanks, that's perfect! I think it would be good to remove the unused space between the new emotes and the current emotes, too. It will likely remain unused in the future otherwise 😅

okliis commented:
Unused space at the bottom is now removed!

Added a few more emotes:
- `Hmm`
- `peepoHey` (gif)
- `LETSGO` (gif)
- `CACTUS` (a popular map asset in TM)
- `PETTHEAUTHOR` (gif)

That makes it a total of 10 new emotes. 😃

codecat commented:
Hey sorry for the delay in merging this, I've been rather busy and haven't had much time lately. I'll take another look at this soon!

okliis commented:
Hey, no worries!

codecat commented:
Thank you! 🎉 Sorry for the late merge 😅
